kalitectl.py

```diff
@@ -219,7 +225,6 @@ def get_pid():
         raise NotRunning(7)  # Unclean shutdown
 
     # TODO: why is the port in django settings!? :) /benjaoming
-    from django.conf import settings
```

Comment:
So the TODO was left in there together with the import statement because we're trying to avoid loading the settings module unnecessarily... but also because we should find a more configurable approach to configuring which port the server runs on.